### ISSUE TITLE
Remove unused `Hasher` aux column

### ISSUE THREAD

**Issue by plafer** [0xPolygonMiden/miden-vm#1385]
According to the [AUX column layout](https://github.com/0xPolygonMiden/miden-vm/blob/107142dc470a8130baf8a900a114e22eb08f2e9e/air/src/trace/mod.rs#L57-L92), we there is currently a hasher aux column. However, we [don't build](https://github.com/0xPolygonMiden/miden-vm/blob/107142dc470a8130baf8a900a114e22eb08f2e9e/processor/src/trace/mod.rs#L221-L226) any hasher aux column.

According to @Al-Kindi-0, this is a leftover from a previous PR where that column was merged into the chiplet bus column.

**Comment by bobbinth** [comment]
Yes, this seems to be wrong - but also I think the chiplets table still has [2 columns](https://github.com/0xPolygonMiden/miden-vm/blob/107142dc470a8130baf8a900a114e22eb08f2e9e/processor/src/chiplets/aux_trace/mod.rs#L58) (even though the comment for this function says that there is only 1 bus column). Theseare:

- A column for the chiplet bus.
- A virtual table column used by the hash chiplet (for the sibling table) and by the Kernel ROM chiplet (for the kernel procedure access table). This column is described in the docs [here](https://0xpolygonmiden.github.io/miden-vm/design/chiplets/main.html#chiplets-virtual-table).

**Comment by Al-Kindi-0** [comment]
Closed by #1694
